backend/utils/send-email.js

```diff
@@ -0,0 +1,101 @@
+const sgMail = require('@sendgrid/mail'); // Twilio SendGrid (https://github.com/sendgrid/sendgrid-nodejs)
+const format = require('date-fns/format');
+const util = require('util');
+
+const insertMerchantId = (origContent, merchantId) => {
+  // const merchantId = "A3L2WCBX4NBSPG"; // This always returns true and is useful for testing
+  const url = `https://primenow.amazon.co.uk/checkout/enter-checkout?merchantId=${merchantId}&ref=pn_sc_ptc_bwr`;
+  const newContent = {
+    subject: origContent.subject,
+    body: origContent.body[1] + ' ' + url
+  };
+
+  return newContent;
+};
```

Comment:
this merchantid is an amazon specific thing, not an email thing. I would suggest undoing Amazon back to how it was since it's unlikely we'll use it any time soon. But if you really want to keep it in here, then we can probs pass the url to it as an arg instead of redefining it here.